bin/midi.ml

```diff
@@ -2,6 +2,12 @@ open! Import
 open Result.Syntax
 module Event = Portmidi.Portmidi_event
 
+let bend_pitch ~bend ~timestamp () =
+  let bend_as_integer = int_of_char bend in
+  let data1 = bend_as_integer land 0b1111111 in
+  let data2 = bend_as_integer lsr 7 in
+    Event.create ~status:'\224' ~data1:data1 ~data2:data2 ~timestamp
```

Comment:
This does not compile as `data1` and `data2` are passed as integers. They should be `char`, you need to convert them again. :-)

Reply:
Sorry, I did not try to compile the code before pushing but I tested the function in `utop` but forgot to convert `data1` and `data2` to char

About setting up my local environment
- I have Ocaml installed in WSL
- The Ocaml extension for VSCode only worked on the first day

Reply:
just pushed the fix

Reply:
> All good then! I enabled the CI to run on your PR so you will get immediate feedback on it next time you push!

It seems you have to approve for the CI to run